### ISSUE TITLE
Comment faire pour faire apparaître l'œ sur le nom d'un auteur ? 

### ISSUE THREAD

**Issue by Rouergue** [EditionsDuRouergue/lerouergue#252]
Au départ, le "o" était séparé du "e" mais l'auteur a protesté. Fatima a donc fait la correction au niveau du nom de l'auteur dans mistral, mais cela apparaît maintenant avec un "?". Comment faire ?

![capture d ecran 2016-03-02 a 12 11 43](https://cloud.githubusercontent.com/assets/10072454/13458838/16378874-e070-11e5-9fd6-273b08e09bff.png)

**Comment by Fchallier** [comment]
J'avais fait une correction au niveau du code mais cela n'a rien changé.. car dans la BDD issu de Mistral il y a bel et bien un "?" à la place du "œ"... 
Contrairement à d'autre auteurs comme  : 
BŒUF 	Georges  - Code : 1111
SŒUR EMMANUELLE 	-  Code 2345
MARBŒUF 	Olivier - Code 19001

**Comment by Rouergue** [comment]
pourtant, cela s'affiche bien chez moi. Est-ce parce que la mise à jour ne s'affiche pas ?

![capture d ecran 2016-03-09 a 10 24 50](https://cloud.githubusercontent.com/assets/10072454/13630754/3dffeb64-e5e1-11e5-9bea-14f69a6ceb40.png)

**Comment by Fchallier** [comment]
Non, j'ai fais la correction à la main après la Mise à jour... Étant donné que l'auteur semble légèrement tatillon :p

**Comment by Rouergue** [comment]
Ok. Mais donc ta correction manuelle ne sera pas "aplatie" à chaque mise à jour du serveur ? C'est bien ça ?

**Comment by Rouergue** [comment]
On se retrouve toujours avec l'O séparé du E sur le site (alors qu'il est bien œ dans mistral)

![Capture d’écran 2020-11-12 à 15 47 44](https://user-images.githubusercontent.com/10072454/98954806-79e0c980-24fe-11eb-8ea4-463808dce9ca.png)

![Capture d’écran 2020-11-12 à 15 45 04](https://user-images.githubusercontent.com/10072454/98954815-7c432380-24fe-11eb-8c6a-72c45396370c.png)

**Comment by Fchallier** [comment]
Il est nécessaire de faire la mise à jour dans Mistral / OLea. 

Voir peut-être avec Bénédicte Abergel ?
